hardware_layer/hardware_arduino/firmware/arduino_node.cpp

```diff
@@ -0,0 +1,183 @@
+#include "ros.h"
+#include <Arduino.h>
+#include <std_msgs/Int32.h>
+#include <std_msgs/Float64.h>
+#include <math.h>
+#include <Wire.h>
+#include "../include/MS5837.h"
+#include "../include/arduino_thrust.h"
+
+#define pwmPinWest 3
```

Comment:
Put pin configuration in a separate header files, just like we have done in [`fourtran`](https://github.com/AUV-IITK/fourtran)